rust-toolchain.toml

```diff
@@ -3,7 +3,7 @@
 # https://github.com/rust-lang/rustup/issues/2686
 # The auto-installation behaviour in rustup will likely be removed:
 # https://github.com/rust-lang/rustup/issues/1397
-channel = "1.70.0"
+channel = "nightly-2023-05-23"
```

Comment:
Why change to nightly here?

Reply:
Just for local testing - to run Intellij test runner - it uses `rust-toolchain.toml`. Will r4evert before I merge. 😄